apps/web/vibes/soul/sections/cart/index.tsx

```diff
@@ -25,18 +27,9 @@ export interface CartLineItem {
   price: string;
 }
 
-interface CartSummary {
-  title?: string;
-  caption?: string;
-  subtotalLabel?: string;
-  subtotal: string | Promise<string>;
-  shippingLabel?: string;
-  shipping?: string;
-  taxLabel?: string;
-  tax?: string | Promise<string>;
-  grandTotalLabel?: string;
-  grandTotal?: string | Promise<string>;
-  ctaLabel?: string;
+interface CartSummaryLineItem {
```

Comment:
A line item is wording usually used in a receipt/order/checkout so I think it makes sense for `cart.lineItems` since those have a quantity and a price, etc. For the summary, I think these are genera UI list items so `CartSummaryItem` is probably fine